### DISCUSSION TITLE
Are there ways to right align the ComboBox in Java?

### DISCUSSION THREAD

**Discussion by KaiwenYangUT** [CSC207-2022F-UofT/course-project-macrosoft#68] [Q&A] [answered]
Hi folks,

I'm currently developing the UpdateOrderStatus feature using JFrame. I'm planning on right align the combobox and adding a text "Order #1" on the left of the combobox, any ideas?

**Comment by xuallenx** [comment, marked as answer]
I think you might find this thread useful 
https://stackoverflow.com/questions/19641404/is-there-any-way-to-right-align-the-text-in-a-jcombobox
good luck! let us know if you encounter any other troubles

**Reply by KaiwenYangUT** [reply]
WOW, that really helps a lot! Thanks Allen!!!!!!
